libs/core/src/lib/vertical-navigation/vertical-navigation.component.ts

```diff
@@ -21,16 +24,86 @@ export class VerticalNavigationComponent implements AfterContentInit {
     @Input()
     condensed = false;
 
+    /** An RxJS Subject that will kill the data stream upon component’s destruction (for unsubscribing)  */
+    private readonly _onDestroy$: Subject<void> = new Subject<void>();
+
+    /** An RxJS Subject that will kill the data stream upon queryList changes (for unsubscribing)  */
+    private readonly _onRefresh$: Subject<void> = new Subject<void>();
+
     /** @hidden */
-    @ContentChildren(ListNavigationItemComponent)
+    @ContentChildren(ListNavigationItemComponent, { descendants: true })
     private _navigationItems: QueryList<ListNavigationItemComponent>;
 
+    /** @hidden
+     * Querylist of list-items in main navigation.
+     */
+    @ContentChildren(ListNavigationItemComponent)
+    private _mainNavigationItems: QueryList<ListNavigationItemComponent>;
```

Comment:
`_directNavigationItems`?

Reply:
I am referring to immediate list-items in main navigation. @platon-rov 

![image](https://user-images.githubusercontent.com/65063487/191998126-dc61fe2a-eeb1-46a7-bcb1-6746a731af88.png)